src/Command/FeedListCommand.php

```diff
@@ -37,20 +39,26 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
-        $showDisabled = $input->getOption('show-disabled');
+        $status = $input->getOption('status');
+        if (!in_array($status, self::SELECTIONS)) {
+            $io->error(sprintf('Status given not valid. Allowed values are (%s)', implode(', ', self::SELECTIONS)));
```

Comment:
Rather that repeating the `implode`-stuff here, you could render the command help:

```suggestion
            $io->error(sprintf('Invalid status: %s', $status));

            // Show how to run this command.
            // https://symfony.com/doc/current/console/calling_commands.html
            $this->getApplication()->doRun(new ArrayInput([
                'command' => $this->getName(),
                '--help'  => true,
            ]), $output);
```